k8s/nginx/container/Dockerfile

```diff
@@ -7,5 +7,5 @@ RUN apt-get update \
     && apt-get clean
 COPY nginx.conf.template /etc/nginx/nginx.conf
 COPY nginx_entrypoint.bash /
-EXPOSE 80 443 27017
+EXPOSE 80 81 443 444 27017
```

Comment:
You seem to be tilting in favor of ports 81 and 444 for WebSocket traffic.

I'm new to WebSocket so I looked around and it seems that it's more standard to use 80 and 443 (i.e. the same as the standard ports for HTTP and HTTPS), e.g. see https://stackoverflow.com/questions/28522857/shall-i-use-websocket-on-ports-other-than-80